TeachingRecordSystem/src/TeachingRecordSystem.Core/Jobs/Scheduling/IBackgroundJobScheduler.cs

```diff
@@ -5,4 +5,8 @@ namespace TeachingRecordSystem.Core.Jobs.Scheduling;
 public interface IBackgroundJobScheduler
 {
     Task Enqueue<T>(Expression<Func<T, Task>> expression) where T : notnull;
+
+    Task<string> EnqueueJob<T>(Expression<Func<T, Task>> expression) where T : notnull;
```

Comment:
Can we amend the existing `Enqueue` instead of adding another method?

Reply:
I didn't want to break everything else that uses that as it will now return a Guid

Reply:
What would break?